runtime/standalone/Cargo.toml

```diff
@@ -35,6 +35,7 @@ pallet-portal-rpc-runtime-api             = { path = "../../pallets/portal/rpc/r
 pallet-xdns                               = { path = "../../pallets/xdns", default-features = false }
 pallet-xdns-rpc-runtime-api               = { path = "../../pallets/xdns/rpc/runtime-api", default-features = false }
 pallet-eth2-finality-verifier             = { git = "https://github.com/t3rn/eth2-light-client", branch = "main", default-features = false }
+#pallet-eth2-finality-verifier             = { path = "../../../eth2-light-client", default-features = false }
```

Comment:
if we will leave it, then merge will do runtime upgrade.
better to add it to our documentation on how to locally use `pallet-eth2-finality-verifier`